Domains/2-DisciplinePhysical/Piping/PipeworkPhysical.ecschema.xml

```diff
@@ -0,0 +1,244 @@
+        <BaseClass>dsys:IDistributionElement</BaseClass>
+        <ECCustomAttributes>
+            <IsMixin xmlns="CoreCustomAttributes.01.00.03">
+                <AppliesToEntityClass>bis:PhysicalElement</AppliesToEntityClass>
+            </IsMixin>
+        </ECCustomAttributes>
+    </ECEntityClass>
+
+    <ECRelationshipClass typeName="PipingSystemGroupsPipingElements" strength="referencing" modifier="None" displayLabel="Distribution System Groups Distribution Elements" description="A relationship used to identify the IPipingElements that are members of a PipingSystem.">
+        <BaseClass>dsys:DistributionSystemGroupsDistributionElements</BaseClass>
+        <Source multiplicity="(0..*)" roleLabel="groups" polymorphic="true">
+            <Class class="PipingSystem"/>
+        </Source>
+        <Target multiplicity="(0..*)" roleLabel="is grouped by" polymorphic="true">
+            <Class class="IPipingElement"/>
+        </Target>
+    </ECRelationshipClass>
+
+    <ECEntityClass typeName="IPipingPhysicalType" modifier="Abstract" displayLabel="Piping Physical Type" description="An interface that can be mixed-into a bis:PhysicalType that defines a shared set of properties whose values vary per-type of IPipingElement rather than per-instance.">
+        <BaseClass>dsys:IDistributionElementType</BaseClass>
+        <ECCustomAttributes>
+            <IsMixin xmlns="CoreCustomAttributes.01.00.03">
+                <AppliesToEntityClass>bis:PhysicalType</AppliesToEntityClass>
+            </IsMixin>
+        </ECCustomAttributes>        
+    </ECEntityClass>
+
+    <ECEntityClass typeName="PipeElement" modifier="Abstract" displayLabel="Pipe Element" description="A bis:PhysicalElement used to typically join two sections of a piping network.">
+        <BaseClass>bis:PhysicalElement</BaseClass>
+        <BaseClass>IPipingElement</BaseClass>
+        <BaseClass>dsys:IDistributionFlowElement</BaseClass>
+    </ECEntityClass>
+
+    <ECEntityClass typeName="PipeElementType" modifier="Abstract" displayLabel="Pipe Element Type" description="Defines a shared set of properties whose values vary per-type of PipeElement rather than per-instance.">
+        <BaseClass>bis:PhysicalType</BaseClass>
+        <BaseClass>IPipingPhysicalType</BaseClass>
+        <ECProperty propertyName="GrossWeight" typeName="double" kindOfQuantity="AECU:WEIGHT" displayLabel="Gross Weight" description="Total Gross Weight of the object without any add-on parts and not taking into account possible processing features (cut-out's, etc.) or openings and recesses. Not including contained matter."/>
+        <ECProperty propertyName="PressureRange" typeName="bis:BoundedDoubleValue" kindOfQuantity="AECU:PRESSURE" displayLabel="Pressure Range" description="Allowable maximum, minimum and typical working pressure (relative to ambient pressure)."/>
```

Comment:
is this a valid definition? pointing to a struct type and setting KOQ?

Reply:
Further discussions led us into proposing a new schema with unit-driven permutations of this Bounded-value concept. It is more compatible with the current implementation at the UX level.